addons/editor/functions/fnc_handleKeyDown.sqf

```diff
@@ -18,6 +18,50 @@
 
 params ["_display", "_keyCode"];
 
+if (_keyCode in actionKeys "curatorPingView" && {!isNil QGVAR(pingTarget)} && {isNil QGVAR(pingViewed)}) exitWith {
```

Comment:
If anything, the nil checks should be together before checking the key, but I think there's a more worthy aspect to look at, that being dropping the lazy eval in this case cuz if a new context really has a 15% performance penalty, such simple checks like nil (that check things that won't be used in the same statement, no less) should be cheaper than the context creation => lazy eval is more trouble than worth.
```suggestion
if (_keyCode in actionKeys "curatorPingView" && !isNil QGVAR(pingTarget) && isNil QGVAR(pingViewed)) exitWith {
```

Reply:
In ZEN code style, we always use lazy eval for conditions.

Reply:
Frankly, that's a bad excuse.